pkg/test/integration/ingest_otlp_test.go

```diff
@@ -143,8 +146,13 @@ func TestIngestOTLP(t *testing.T) {
 
 					expectedBytes, err := os.ReadFile(metric.expectedJsonPath)
 					require.NoError(t, err)
+					var expected strprofile.CompactProfile
+					assert.NoError(t, json.Unmarshal(expectedBytes, &expected))
+					strprofile.SortProfileSamples(expected)
```

Comment:
This sorting likely fixes the issue I saw with go 1.24 
https://github.com/grafana/pyroscope/actions/runs/14830299218/job/42561162180
Thanks :heart: